### PR TITLE
feat: Adjust panels spacing

### LINKED ISSUE #23: Unify blank states whitespacing
Spacing between tabs and "There is no data available" messages is inconsistent.

### LINKED ISSUE #394: Adjust panels spacing
Adjust panels spacing according to the new Figma designs.

TBD if panels should have a dedicated component for the header.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #394 
fixes #23

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] abstract panel components
- [x] unify blank panels spacing
- [x] adjust panel spacing by visual material

### CODE REVIEW

**Comment by janmichek** [comment]
Followup, I didn't dare because of many changes https://github.com/aeternity/aescan/issues/449

File: src/components/AccountDetailsPanel.vue [hunk 1]
```diff
@@ -142,33 +138,26 @@ const sanitizedPrice = computed(() =>
 
 <style scoped>
 .account-details-panel {
-  padding: var(--space-4) var(--space-1) var(--space-3);
-  margin-bottom: var(--space-7);
+  &__table-header {
```
Comment (janmichek) [inline src/components/AccountDetailsPanel.vue:141]: Styles were sorted and ordered, which uncover a lot of dead code

File: src/components/AppSelect.vue [hunk 2]
```diff
@@ -83,13 +83,17 @@ const selectedValue = useVModel(props, 'modelValue', emit)
   &__tags {
     border-radius: 8px;
     border-color: var(--color-midnight);
+    border-width: 1.5px;
```
Comment (janmichek) [inline src/components/AppSelect.vue:86]: updated minor select tyles

File: src/components/AppTabs.vue [hunk 3]
```diff
@@ -73,9 +73,10 @@ function selectTab(tabIndex) {
     cursor: pointer;
     padding: var(--space-1);
     background: var(--color-midnight-15);
-    color: var(--color-gray);
+    color: var(--color-midnight-55);
```
Comment (janmichek) [inline src/components/AppTabs.vue:76]: updated colors of tabs

File: src/components/AuctionsPanel.vue [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <app-panel class="auctions-panel">
     <panel-header
       level="h3"
-      title="Auctions ending soon"
+      title="AUCTIONS ENDING SOON"
```
Comment (janmichek) [inline src/components/AuctionsPanel.vue:5]: Reverted last changes in dashboard headings after consultation

File: src/components/DashboardStateChannelsPanel.vue [hunk 5]
```diff
@@ -4,7 +4,8 @@
       level="h3"
       title="State Channels"
       icon-name="state-channel"
-      show-all-link="/state-channels">
+      show-all-link="/state-channels"
+      class="dashboard-state-channels-panel__panel-header">
```
Comment (michele-franchi) [inline src/components/DashboardStateChannelsPanel.vue:8]: I think this class does not exist

File: src/components/StatsTile.vue [hunk 6]
```diff
@@ -41,6 +41,7 @@ defineProps({
   display: flex;
   flex-direction: row;
   word-break: break-word;
+  padding: 8px 0;
```
Comment (michele-franchi) [inline src/components/StatsTile.vue:44]: Shouldn't `space-1` be used?

File: src/components/TransactionCellNameRevokeTx.vue [hunk 7]
```diff
@@ -4,7 +4,9 @@
     :link-to="`/accounts/${transactionData.accountId}`"/>
   <transaction-arrow-right-icon/>
   <span class="u-ellipsis transaction-cell-name-revoke-tx__name">
-    <app-link :to="`/names/${transactionData.name}`">
+    <app-link
+      size="sm"
```
Comment (michele-franchi) [inline src/components/TransactionCellNameRevokeTx.vue:8]: We don't have sizes on app-link

File: src/pages/keyblocks/[id].vue [hunk 8]
```diff
@@ -49,3 +50,12 @@ await useAsyncData(async() => {
   return true
 })
 </script>
+
+<style scoped>
+.keyblocks_keyblock-details-panel {
```
Comment (michele-franchi) [inline src/pages/keyblocks/[id].vue:55]: Maybe `keyblock-details__keyblock-details-panel` is more precise?

File: src/pages/oracles/index.vue [hunk 9]
```diff
@@ -16,7 +16,9 @@
     </template>
   </page-header>
 
-  <oracles-panel v-if="!isLoading"/>
+  <oracles-panel
+    v-if="!isLoading"
+    class="oracles__oracle-panel"/>
```
Comment (michele-franchi) [inline src/pages/oracles/index.vue:21]: This class is not defined

File: src/pages/microblocks/[id].vue [hunk 10]
```diff
@@ -41,3 +47,11 @@ await useAsyncData(async() => {
   return true
 })
 </script>
+<style scoped>
+.microblock-details-panel {
```
Comment (michele-franchi) [inline src/pages/microblocks/[id].vue:51]: Maybe `.microblock-details__microblock-details-panel` is  more precise?

File: src/pages/tokens/index.vue [hunk 11]
```diff
@@ -10,7 +10,9 @@
       {{ tokensHints.token }}
     </template>
   </page-header>
-  <tokens-panel v-if="!isLoading"/>
+  <tokens-panel
+    v-if="!isLoading"
+    class="tokens__tokens-panel"/>
```
Comment (michele-franchi) [inline src/pages/tokens/index.vue:15]: This class is not defined

**Review by michele-franchi — CHANGES_REQUESTED** [review]
Impressive work, really well done! 👏 

I left some comments, could you have a look at them?

I personally don't like 1.5px borders and I don't understand where these come from, I checked Figma and it's always 1px. I didn't comment in the code because there are a lot of places but personally I would keep 1px everywhere, as It was before.

File: src/pages/tokens/index.vue (hunk 11 above)
Comment (janmichek) [reply src/pages/tokens/index.vue:15]: Right, removed

File: src/pages/microblocks/[id].vue (hunk 10 above)
Comment (janmichek) [reply src/pages/microblocks/[id].vue]: yes, fixed. Thanks for the consistency check

File: src/pages/oracles/index.vue (hunk 9 above)
Comment (janmichek) [reply src/pages/oracles/index.vue]: Removed

File: src/pages/keyblocks/[id].vue (hunk 8 above)
Comment (janmichek) [reply src/pages/keyblocks/[id].vue]: Absolutely, ser. Fixed

File: src/components/StatsTile.vue (hunk 6 above)
Comment (janmichek) [reply src/components/StatsTile.vue]: yep, fixed

File: src/components/DashboardStateChannelsPanel.vue (hunk 5 above)
Comment (janmichek) [reply src/components/DashboardStateChannelsPanel.vue]: Nice catch removed

**Comment by janmichek** [comment]
> Impressive work, really well done! 👏
> 
> I left some comments, could you have a look at them?
> 
> I personally don't like 1.5px borders and I don't understand where these come from, I checked Figma and it's always 1px. I didn't comment in the code because there are a lot of places but personally I would keep 1px everywhere, as It was before.

Well, this is tricky. In code it is 1px, but in every zoomed scale looks like 1.5px,
It's surprising to me that such a mature tool as Figma cannot render correctly. I have been working with Figma several times before and I didn't face such render issues. Can you provide some feedback @danilosierrac ?

**Comment by janmichek** [comment]
@michele-franchi I don't think that is how Figma works. I didn't have that issue before. However, I fixed it. All borders aligned to 1px

**Comment by michele-franchi** [comment]
@janmichek for example for fonts:
https://forum.figma.com/t/why-does-a-font-weight-in-figma-seem-lighter-than-the-same-weight-in-the-browser/2207

>Figma uses its own font rendering because each device renders fonts differently depending on the device, OS type and version, browser and its version, system settings, browser settings, display and maybe some other factors. Figma has to use one font rendering for all because it’s a collaborative tool with one source of truth.

**Review by michele-franchi — APPROVED** [review]
Very good job 🚀

File: src/styles/elements/_table.css [hunk 12]
```diff
@@ -3,37 +3,44 @@ table {
...
 
 
   @media (--desktop) {
     font-size: 16px;
-    line-height: 24px;
-    vertical-align: middle;
-    padding: var(--space-2) 0 var(--space-1);
+    line-height: 18px;
+    vertical-align: top;
+    padding: 10px 0 8px;
```
Comment (lukeromanowicz) [inline src/styles/elements/_table.css:21]: We could use --space-1 here

File: src/styles/elements/_table.css [hunk 13]
```diff
@@ -3,37 +3,44 @@ table {
...
-  padding: var(--space-1) 0;
+  padding: 7px 0;
   text-align: left;
-  border-bottom: 1px solid var(--color-midnight-15);
+  border-bottom: 1px solid var(--color-midnight-25);
   vertical-align: middle;
 
   @media (--desktop) {
-    padding: var(--space-1) var(--space-0);
+    padding: 10px var(--space-0) 8px;
```
Comment (lukeromanowicz) [inline src/styles/elements/_table.css:34]: I would use --space-1 here

File: src/styles/elements/_table.css (hunk 12 above)
Comment (janmichek) [reply src/styles/elements/_table.css]: fixed

**Comment by janmichek** [comment]
@Liubov-crypto Could you please go through the whole app and check if anything is obviously broken from UI side. Please don't compare to design material, as there are more misalignments. Thank you

**Review by Liubov-crypto — APPROVED** [review]
I found that the info is duplicated on Tokens details page, but I think this is middleware issue:
dubb


The rest, LGTM.

**Comment by janmichek** [comment]
> I found that the info is duplicated on Tokens details page, but I think this is middleware issue: dubb
> 
> The rest, LGTM.

Rigth, that's a know thing. Thanks for testing

**Comment by Liubov-crypto** [comment]
> > I found that the info is duplicated on Tokens details page, but I think this is middleware issue: dubb
> > The rest, LGTM.
> 
> Rigth, that's a know thing. Thanks for testing

You are welcome! :)

**Pull request merged**